lib/cisco_node_utils/feature.rb

```diff
@@ -80,7 +80,12 @@ def self.nv_overlay_enabled?
 
     # ---------------------------
     def self.nv_overlay_evpn_enable
-      return if nv_overlay_evpn_enabled?
+      begin
+        return if nv_overlay_evpn_enabled? && fabric_forwarding_enabled?
```

Comment:
Don't check `fabric_forwarding_enabled?` since that will be false on the platforms that don't support it.